Engine/source/console/compiledEval.cpp

```diff
@@ -435,11 +435,12 @@ static void setFieldComponent( SimObject* object, StringTableEntry field, const
 
 ConsoleValueRef CodeBlock::exec(U32 ip, const char *functionName, Namespace *thisNamespace, U32 argc, ConsoleValueRef *argv, bool noCalls, StringTableEntry packageName, S32 setFrame)
 {
+/*
 #ifdef TORQUE_DEBUG
    U32 stackStart = STR.mStartStackSize;
    U32 consoleStackStart = CSTK.mStackPos;
 #endif
-
+*/
```

Comment:
Probably should just use another define for this instead of commenting the whole thing out - eew!